jwst/datamodels/schemas/extract1dimage.schema.yaml

```diff
@@ -22,7 +22,7 @@ allOf:
       type: integer
       fits_keyword: SPORDER
       fits_hdu: SCI
-    dispersion_axis:
+    dispersion_direction:
```

Comment:
What's the reason for this change?  I think `dispersion_axis` is much more descriptive.  The value can be 1 or 2, which is the FITS 1-indexed axis of the dispersion.  It's not really a direction.  It's an axis.

Reply:
Ah, I see it is `dispersion_direction` in `wcsinfo`.  I would suggest changing it there (and where ever it used in the code) to `disperision_axis`.

Reply:
When it was first implemented in both the KWD and our schema (wcsinfo) it was entered as `dispersion_direction`. When it then got added later to other schema, it was done using a different name. So this change is simply an effort to make them all consistent. It's easiest for us to change the inconsistencies in our schemas, rather than having the KWD change to match us.